pontoon/static/js/tags/admin/search.js

```diff
@@ -0,0 +1,47 @@
+
+import React from 'react';
+
+import {Columns} from 'widgets/columns';
+
+
+export default class TagResourceSearch extends React.PureComponent {
+
+    get columns () {
+        return [[this.renderSearchInput(), 3],
+                [this.renderSearchSelect(), 2]];
+    }
+
+    handleChange = (evt) => {
+        switch (evt.target.nodeName) {
+        case 'INPUT':
+            return this.props.handleSearchChange(
+                {search: evt.target.value});
+        case 'SELECT':
+            return this.props.handleSearchChange(
+                {type: evt.target.value});
+        }
+    }
```

Comment:
You can make this more generic and more explicit at the same time by adding a `name` attribute to your elements. See the official documentation: https://reactjs.org/docs/forms.html#handling-multiple-inputs

```javascript
handleInputChange(event) {
    const target = event.target;
    this.props.handleSearchChange({
        [target.name]: target.value,
    });
}
```